### ISSUE TITLE
Running version version installed version

### ISSUE THREAD

**Issue by wimpers** [openvstorage/openvstorage-health-check#362]
ALBA allows to ask processes to define which version they are running of ALBA. This should be compared with the installed version as it might indicate that an alba component needs to be restarted or that something is wrong.

**Comment by JeffreyDevloo** [comment]
So a check which checks if the components running version is behind the installed version?

**Comment by wimpers** [comment]
Yes or report the running version instead of the installed version if not done already.

**Comment by wimpers** [comment]
Lets do both:
* report running version 
* check whether installed version and running version are identical -> warning
